.github/workflows/pkg-vc.yml

```
@@ -0,0 +1,39 @@
+permissions:
+    pull-requests: write
+    contents: read
+
+jobs:
+    publish:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v4
+            - name: Use Node.js
+              uses: actions/setup-node@v2
```

actionlint (1.7.7)

the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)